state/runtime/allowlist/genesis.go

```diff
@@ -0,0 +1,51 @@
+package allowlist
+
+import (
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+func ApplyGenesisAllocs(chain *chain.Genesis, allowListAddr types.Address, config *chain.AllowListConfig) {
+	allocList := &AllowList{
+		addr:  allowListAddr,
+		state: &genesisState{chain},
+	}
+
+	// enabled addr
+	for _, addr := range config.EnabledAddresses {
+		allocList.SetRole(addr, EnabledRole)
+	}
+
+	// admin addr
+	for _, addr := range config.AdminAddresses {
+		allocList.SetRole(addr, AdminRole)
+	}
+}
+
+type genesisState struct {
+	chain *chain.Genesis
+}
+
+func (g *genesisState) SetState(addr types.Address, key, value types.Hash) {
+	alloc, ok := g.chain.Alloc[addr]
+	if !ok {
+		alloc = &chain.GenesisAccount{}
+		g.chain.Alloc[addr] = alloc
+	}
+
+	// initialize a balance of at least 1 since otherwise
+	// the evm understand that this account is empty
+	alloc.Balance = big.NewInt(1)
+
+	if alloc.Storage == nil {
+		alloc.Storage = map[types.Hash]types.Hash{}
+	}
+
+	alloc.Storage[key] = value
+}
+
+func (g *genesisState) GetStorage(addr types.Address, key types.Hash) types.Hash {
+	panic("not used")
+}
```

Comment:
Maybe to consider this to be no-op implementation, just returning `types.ZeroHash` all the time. Is it a better idea? I know that intention 
here is to prevent somebody to call this function, but again, not sure is the panic right way.

Would it harm if we were returning `Storage` from the `alloc` (if I'm correct there)?
So it would go like this:

```go
if alloc, exists := g.chain.Alloc[addr];exists {
    return alloc.Storage[key]
}

return types.ZeroHash
```

Anyways if nothing of afore-mentioned is not appropriate, then rebase PR on `develop`, since Goran introduced custom linting rule, which prevents explicit panicing, so suppress it here.

Reply:
It would not really affect it because it cannot ever be called at all. But given Goran work on the linting rules, I will return an empty `types.Hash` and comment out that the function will never be called.